src/include/daos_srv/control.h

```diff
@@ -90,6 +101,43 @@ nvme_str2state(char *state)
 	return NVME_DEV_STATE_INVALID;
 }
 
+static inline char *
+led_state2str(int state)
+{
+	/** VMD LED states */
+	if (BIT_SET(state, VMD_LED_STATE_OFF))
+		return "OFF";
+
+	if (BIT_SET(state, VMD_LED_STATE_IDENTIFY))
```

Comment:
should "fault" not be checked for first? or is there a check somewhere to make sure only one bit is ever set in the state?